### PR TITLE
Add websocket feature

### PR DESCRIPTION
**Pull request by ctrlaltca**
## Related Ticket(s)
- Related #3537

## What will change with this Pull Request?
A new "websocket" feature will be exposed by the server, the client and the webclient.
A server could make the feature mandatory and inform users that they need to update their trice to be able to connect in the future.

### CODE REVIEW

**Comment by ZeldaZach** [comment]
I ask you also add a flag, `2.7.0_min_version` to the config as that will be useful in the near future and I'd rather do one PR modifying these files

**Comment by ctrlaltca** [comment]
@ZeldaZach `2.7.0_min_version` added.

File: webclient/webclient.js [hunk 1]
```diff
@@ -168,8 +168,9 @@ var WebClient = {
         "feature_set",
         "room_chat_history",
         "client_warnings",
-        "2.6.1_min_version",
         "websocket",
+        "2.6.1_min_version",
+        "2.7.0_min_version",
         /* unimplemented features */
```
Comment (tooomm) [inline webclient/webclient.js]: Unimplemented features? hmm?
We have forgot pw, and idle client timer?

Comment (ctrlaltca) [reply webclient/webclient.js]: I don't get the question, what do you mean?

Comment (tooomm) [reply webclient/webclient.js]: It looks like the list is somehow separated by a comment.
But the divider makes no sense to me:
```
      "clientfeatures" : [
        "client_id",
        "client_ver",
        "feature_set",
        "room_chat_history",
        "client_warnings",
        /* unimplemented features */
        "forgot_password",
        "idle_client",
        "mod_log_lookup",
        "user_ban_history"
]
```

See https://github.com/Cockatrice/Cockatrice/blob/master/webclient/webclient.js#L165-L176

Comment (tooomm) [reply webclient/webclient.js]: Scrape that, it's the webclient.... 🙈

Comment (ctrlaltca) [reply webclient/webclient.js]: The webclient only implements the features before the comment.
The others are still included only to be able to connect to servers requiring/declaring them.

**Pull request merged**